components/statsTracking/fn_addStatsEventHandlersToClass.sqf

```diff
@@ -0,0 +1,47 @@
+// Adds event handlers to the given unit, to track various stats for viewing on the debriefing.
+// These event handlers are local to the client, and the data they collect is relevant only 
+// to the client, so we don't need to sync the data across the network.
+params ["_unit"];
+
+if !(_unit isEqualTo player) exitWith {};
+
+if (hasInterface) then {
+	["ace_firedPlayer", {
+		params ["_unit", "_weapon", "_muzzle", "_mode", "_ammo", "_magazine", "_projectile"];
+		if (_unit == player) then {
+			_currentShots = missionNamespace getVariable ["cafe_playerShots", createHashMap];
```

Comment:
Prefer to create this variable in a `global.sqf` file so that these get/sets aren't necessary.